PacService.ERROR_0001_ROLE_CREATION_FAILED=\u00c9chec de la cr\u00e9ation du r\u00f4le : {0}.
PacService.ERROR_0002_NO_CREATE_ROLE_PERMISSION=Autorisations insuffisantes pour cr\u00e9er le r\u00f4le : {0}.
PacService.ERROR_0003_ROLE_CREATION_FAILED_NON_EXISTING_USER=\u00c9chec de la cr\u00e9ation du r\u00f4le \u00ab {0} \u00bb. Tentative d'ajout de l'utilisateur inexistant \u00ab {0} \u00bb au r\u00f4le.
PacService.ERROR_0004_USER_CREATION_FAILED=\u00c9chec de la cr\u00e9ation de l'utilisateur : {0}.
PacService.ERROR_0005_NO_CREATE_USER_PERMISSION=Autorisations insuffisantes pour cr\u00e9er l'utilisateur : {0}.
PacService.ERROR_0006_USER_CREATION_FAILED_NON_EXISTING_USER=\u00c9chec de la cr\u00e9ation de l'utilisateur \u00ab {0} \u00bb. Tentative d'ajout de l'utilisateur inexistant \u00ab {0} \u00bb au r\u00f4le du nouvel utilisateur.
PacService.ERROR_0049_ROLE_ALREADY_EXIST=Le r\u00f4le existe d\u00e9j\u00e0 \u00ab {0} \u00bb
PacService.ERROR_0050_UNRECOGNIZED_ROLE_CREATION=La cr\u00e9ation du r\u00f4le a \u00e9chou\u00e9 en raison d'une erreur non reconnue {0}
PacService.ERROR_0051_USER_ALREADY_EXIST=L'utilisateur existe d\u00e9j\u00e0 \u00ab {0} \u00bb
PacService.ERROR_0052_UNRECOGNIZED_USER_CREATION=La cr\u00e9ation de l'utilisateur a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0053_UNRECOGNIZED_ROLE_DELETION=La suppression du r\u00f4le a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0054_UNRECOGNIZED_USER_DELETION=La suppression de l'utilisateur a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0055_UNRECOGNIZED_ROLE_RETRIEVAL=La r\u00e9cup\u00e9ration du r\u00f4le a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0056_UNRECOGNIZED_USER_RETRIEVAL=La r\u00e9cup\u00e9ration de l'utilisateur a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0057_UNRECOGNIZED_ROLES_RETRIEVAL=La r\u00e9cup\u00e9ration des r\u00f4les a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0058_UNRECOGNIZED_USERS_RETRIEVAL=La r\u00e9cup\u00e9ration des utilisateurs a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0059_UNRECOGNIZED_ROLES_UPDATE=La mise \u00e0 jour du r\u00f4le a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
PacService.ERROR_0060_UNRECOGNIZED_USERS_UPDATE=La mise \u00e0 jour de l'utilisateur a \u00e9chou\u00e9 en raison d'une erreur non reconnue. {0}
#
PacService.ERROR_0007_DATASOURCE_CREATION_FAILED=La cr\u00e9ation de la source de donn\u00e9es a \u00e9chou\u00e9 : {0}.
PacService.ERROR_0008_NO_CREATE_DATASOURCE_PERMISSION=Autorisations insuffisantes pour cr\u00e9er la source de donn\u00e9es : {0}.
PacService.ERROR_0009_DATASOURCE_ALREADY_EXIST=La source de donn\u00e9es existe d\u00e9j\u00e0 : {0}.
#
PacService.ERROR_0010_ROLE_DELETION_FAILED_NO_ROLE=La suppression du r\u00f4le a \u00e9chou\u00e9, car le r\u00f4le n'existe pas : {0}
PacService.ERROR_0011_ROLE_DELETION_FAILED=\u00c9chec de suppression du r\u00f4le : {0}
PacService.ERROR_0012_ROLE_DELETION_FAILED_NO_PERMISSION=Autorisations insuffisantes pour supprimer le r\u00f4le : {0}.
#
PacService.ERROR_0013_USER_DELETION_FAILED_NO_USER=La suppression de l'utilisateur a \u00e9chou\u00e9, car l'utilisateur n'existe pas : {0}
PacService.ERROR_0014_USER_DELETION_FAILED=\u00c9chec de suppression de l'utilisateur : {0}
PacService.ERROR_0015_USER_DELETION_FAILED_NO_PERMISSION=Autorisations insuffisantes pour supprimer l'utilisateur : {0}.
#
PacService.ERROR_0016_DATASOURCE_DELETION_FAILED_NO_DATASOURCE=\u00c9chec de la suppression de la source de donn\u00e9es, car {0} n'existe pas {1}
PacService.ERROR_0017_DATASOURCE_DELETION_FAILED=\u00c9chec de la suppression de la source de donn\u00e9es : {0}.
PacService.ERROR_0018_DATASOURCE_DELETION_FAILED_NO_PERMISSION=Autorisations insuffisantes pour supprimer la source de donn\u00e9es : {0}.
#
PacService.ERROR_0019_DATASOURCE_UPDATE_FAILED=\u00c9chec lors de la mise \u00e0 jour de la source de donn\u00e9es : {0}
PacService.ERROR_0020_DATASOURCE_UPDATE_FAILED_NO_PERMISSION=Autorisations insuffisantes pour mettre \u00e0 jour la source de donn\u00e9es : {0}.
PacService.ERROR_0021_DATASOURCE_UPDATE_FAILED_DOES_NOT_EXIST=La source de donn\u00e9es \u00e0 mettre \u00e0 jour n'existe pas : {0}
# 
PacService.ERROR_0022_FAILED_TO_FIND_DATASOURCE=\u00c9chec lors de la recherche d'une source de donn\u00e9es : {0}
PacService.ERROR_0023_FAILED_TO_GET_DATASDOURCE=Impossible d'obtenir une source de donn\u00e9es : {0}
PacService.ERROR_0024_CONNECTION_ATTEMPT_FAILED=La tentative de connexion a \u00e9chou\u00e9, aucune classe de pilote n'est disponible : {0}
PacService.ERROR_0025_UNABLE_TO_CONNECT=La tentative de connexion a \u00e9chou\u00e9 : {0}
PacService.ERROR_0026_DRIVER_NOT_FOUND_IN_CLASSPATH=La tentative de connexion a \u00e9chou\u00e9, la classe de pilote {0} ne se trouve pas dans le Classpath.
PacService.ERROR_0027_UNABLE_TO_INSTANCE_DRIVER=La tentative de connexion a \u00e9chou\u00e9, impossible de cr\u00e9er l'instance de classe de pilote : {0}
PacService.ERROR_0028_QUERY_NOT_VALID=La configuration de source de donn\u00e9es ne contient pas de requ\u00eate de validation.
PacService.ERROR_0029_QUERY_VALIDATION_FAILED=\u00c9chec de validation de la requ\u00eate. Cause : {0}
PacService.ERROR_0030_FAILED_TO_FIND_ROLE=\u00c9chec lors de la recherche du r\u00f4le : {0}
PacService.ERROR_0031_FAILED_TO_GET_ROLE_NAME=Impossible d'obtenir les noms de r\u00f4le.
PacService.ERROR_0032_FAILED_TO_FIND_USER=\u00c9chec lors de la recherche de l'utilisateur : {0}
PacService.ERROR_0033_FAILED_TO_GET_USER_NAME=Impossible d'obtenir les noms d'utilisateur.
PacService.ERROR_0034_ROLE_UPDATE_FAILED=\u00c9chec lors de la mise \u00e0 jour du r\u00f4le : {0}
PacService.ERROR_0035_ROLE_UPDATE_FAILED_NO_PERMISSION=Autorisations insuffisantes pour mettre \u00e0 jour le r\u00f4le : {0}.
PacService.ERROR_0036_ROLE_UPDATE_FAILED_DOES_NOT_EXIST=Le r\u00f4le \u00e0 mettre \u00e0 jour n'existe pas : {0}
PacService.ERROR_0037_ROLE_UPDATE_FAILED_USER_DOES_NOT_EXIST=L'utilisateur \u00e0 ajouter au r\u00f4le {0} n'existe pas : {1}
PacService.ERROR_0038_USER_UPDATE_FAILED=\u00c9chec lors de la mise \u00e0 jour de l'utilisateur : {0}
PacService.ERROR_0061_USER_UPDATE_FAILED_NO_PERMISSION=Autorisations insuffisantes pour mettre \u00e0 jour l'utilisateur : {0}.
PacService.ERROR_0039_USER_UPDATE_FAILED_DOES_NOT_EXIST=L'utilisateur \u00e0 mettre \u00e0 jour n'existe pas : {0}
PacService.ERROR_0040_USER_UPDATE_FAILED_ROLE_DOES_NOT_EXIST=Le r\u00f4le \u00e0 ajouter \u00e0 l'utilisateur {0} n'existe pas : {1}
PacService.ERROR_0041_FAILED_TO_GET_BISERVER_BASE_URL=Impossible d'obtenir l'url de base du serveur BI
PacService.ERROR_0042_FAILED_TO_GET_BISERVER_CONTEXT_PATH=Impossible d'obtenir le chemin du contexte du serveur BI
PacService.ERROR_0043_TXN_ROLLBACK_FAILED=La restauration de la transaction a \u00e9chou\u00e9.
PacService.ERROR_0044_CLOSE_PROPS_STREAM_FAILED=\u00c9chec de la fermeture du flux d'entr\u00e9e associ\u00e9 au fichier : {0}
PacService.ERROR_0045_THREAD_SCHEDULING_FAILED=\u00c9chec de la planification du thread de statut du serveur BI.
PacService.ERROR_0046_SERVICE_INITIALIZATION_FAILED=\u00c9chec d'initialisation de la console d'administration. {0}
PacService.ERROR_0047_IO_ERROR=Erreur d'E/S lors du chargement de {0}
PacService.ERROR_0048_ROLLBACK_FAILED=\u00c9chec de restauration de la transaction.
PacService.ERROR_0062_UNABLE_TO_REFRESH_HIBERNATE=Impossible d'actualiser Hibernate
PacService.ACTION_COMPLETE=Action termin\u00e9e.
#
StopJettyServer.ERROR_0002_LOAD_PROPS_FAILED=\u00c9chec du chargement du fichier de propri\u00e9t\u00e9s : {0}
StopJettyServer.ERROR_0001_OPEN_PROPS_FAILED=\u00c9chec de l'ouverture du fichier de propri\u00e9t\u00e9s : {0}
StopJettyServer.ERROR_0003_UNKNOWN_HOST=H\u00f4te inconnu : {0}
#
ConsoleProperties.ERROR_0002_LOAD_PROPS_FAILED=\u00c9chec du chargement du fichier de propri\u00e9t\u00e9s : {0}
ConsoleProperties.ERROR_0001_OPEN_PROPS_FAILED=\u00c9chec de l'ouverture du fichier de propri\u00e9t\u00e9s : {0}
#
#
XmlSerializer.stateNormal=Normal
XmlSerializer.stateSuspended=Interrompu
XmlSerializer.stateComplete=Termin\u00e9
XmlSerializer.stateError=Erreur
XmlSerializer.stateBlocked=Bloqu\u00e9
XmlSerializer.stateNone=Aucun
#
ThreadSafeHttpClient.ERROR_0002_INVALID_HTTP_METHOD_TYPE=Type de m\u00e9thode http non valide : {0}.
ThreadSafeHttpClient.ERROR_0001_CLIENT_REQUEST_FAILED=La demande du client pour l'URI \u00ab {0} \u00bb a \u00e9chou\u00e9. Raison de l'\u00e9chec : {1}.
ThreadSafeHttpClient.ERROR_0003_AUTHORIZATION_FAILED=Impossible de se connecter au serveur BI. V\u00e9rifiez les identifiants de l'utilisateur et la configuration XML des param\u00e8tres Web.
JettyServer.CONSOLE_STARTING=La console est en cours de d\u00e9marrage
JettyServer.WAITING_TO_HALT=D\u00e9lai d'attente avant d'arr\u00eater la console
JettyServer.HALTING=Arr\u00eat de la console
JettyServer.STOPPING=Arr\u00eat de la console. Veuillez patienter
JettyServer.CONSOLE_STARTED=La console est maintenant d\u00e9marr\u00e9e. Il est possible d'y acc\u00e9der \u00e0 l'aide de {0} ou {1}
JettyServer.ERROR_0001_UNABLE_START_SERVER=erreur lors du d\u00e9marrage du serveur
JettyServer.ERROR_0002_IO_ERROR=Erreur d'E/S : {0}
#
JdbcDriverDiscoveryService.ERROR_0001_NO_JDBC_DRIVER_PATH_SPECIFIED=Aucun chemin d'acc\u00e8s du pilote jbdb sp\u00e9cifi\u00e9
JdbcDriverDiscoveryService.ERROR_0002_NO_DRIVERS_FOUND_IN_JDBC_DRIVER_PATH=Aucun pilote jdbc trouv\u00e9 dans le chemin d'acc\u00e8s sp\u00e9cifi\u00e9
#
AppConfigProperties.ERROR_0001_INCORRECT_SOLUTION_PATH=Impossible de lire le fichier : {0} {1}. Veuillez vous assurer que le chemin d'acc\u00e8s de la solution est correct.
AppConfigProperties.ERROR_0002_INCORRECT_WAR_PATH=Impossible de lire le fichier : {0} {1}. Veuillez vous assurer que le chemin d'acc\u00e8s war est correct.
AppConfigProperties.ERROR_0003_FAILED_TO_CLOSE_STREAM=\u00c9chec de la fermeture du flux associ\u00e9 \u00e0 : {0} {1}
AppConfigProperties.ERROR_0004_UNABLE_TO_READ_FILE=Impossible de lire le fichier : {0}
AppConfigProperties.ERROR_0005_CLOSE_STREAM_FAILED=\u00c9chec de la fermeture du flux associ\u00e9 \u00e0 : {0}
AppConfigProperties.ERROR_0006_UNABLE_TO_GET_PASSWORD_SERVICE=Impossible d'obtenir l'impl\u00e9mentation du service de mot de passe. Utilisation du service de mot de passe par d\u00e9faut {0}
AppConfigProperties.ERROR_0007_WAR_PATH_NOT_CONFIGURED=Le chemin d'acc\u00e8s war Pentaho n'est pas correctement configur\u00e9. Veuillez corriger le chemin au format console.xml et red\u00e9marrer la console.
AppConfigProperties.ERROR_0008_SOLUTION_PATH_NOT_CONFIGURED=Le chemin d'acc\u00e8s de la solution Pentaho n'est pas correctement configur\u00e9. Veuillez corriger le chemin au format console.xml et red\u00e9marrer la console.
AppConfigProperties.ERROR_0009_UNABLE_TO_GET_ABSOLUTE_PATH=Impossible d'obtenir le chemin d'acc\u00e8s absolu du fichier. {0} Cause : {1}
AppConfigProperties.ERROR_0010_UNABLE_TO_GET_PASSWORD_ENCODER=impossible de cr\u00e9er l'instance d'encodeur de mot de passe
AppConfigProperties.USING_DEFAULT_WAR_PATH=Le chemin d'acc\u00e8s war Pentaho n'\u00e9tait pas correctement configur\u00e9 au format console.xml. Utilisation du chemin d'acc\u00e8s war par d\u00e9faut. {0}
AppConfigProperties.USING_DEFAULT_SOLUTION_PATH=Le chemin d'acc\u00e8s de la solution Pentaho n'\u00e9tait pas correctement configur\u00e9 au format console.xml. Utilisation du chemin d'acc\u00e8s de la solution par d\u00e9faut. {0}
DAOFactory.WARN_0001_UNABLE_TO_INITIALIZE_USER_ROLE_DAO=Impossible d'initialiser
#
#
JettyServer.ERROR_0001_KEY_STORE_MUST_BE_SET=KeyStore ou la propri\u00e9t\u00e9 syst\u00e8me javax.net.ssl.keyStore doit \u00eatre d\u00e9fini
JettyServer.ERROR_0002_KEY_STORE_PASSWORD_MUST_BE_SET=keyStorePassWord ou la propri\u00e9t\u00e9 syst\u00e8me javax.net.ssl.keyStorePassword doit \u00eatre d\u00e9fini
#
HibernateSessionFactory.ERROR_0001_UNKNOWN_CONFIGURATION=Configuration inconnue : {0}
HibernateSessionFactory.ERROR_0002_MISSING_MANAGED_CONFIGURATION=Hibernate est configur\u00e9 pour \u00eatre g\u00e9r\u00e9 et {0} est manquant ou null
HibernateSessionFactory.ERROR_0003_UNABLE_TO_CREATE_SESSION_FACTORY=Impossible de cr\u00e9er la fabrique de session : {0}
HibernateSessionFactory.ERROR_0004_UNABLE_TO_REBUILD_SESSION_FACTORY=Impossible de reg\u00e9n\u00e9rer la fabrique de session : {0}
HibernateSessionFactory.ERROR_0005_MISSING_CONFIGURATION_FILE=Le fichier de configuration Hibernate est manquant : {0}
#
PentahoXml.ERROR_0001_INVALID_ROOT_ELEMENT=\u00c9l\u00e9ment racine non valide.
HibernateSettingsXml.ERROR_0001_INVALID_ROOT_ELEMENT=\u00c9l\u00e9ment racine non valide.
PentahoObjectsConfig.ERROR_0001_INVALID_ROOT_ELEMENT=\u00c9l\u00e9ment racine non valide.
WebXml.ERROR_0001_INVALID_ROOT_ELEMENT=\u00c9l\u00e9ment racine non valide.
GoogleMapsConfig.ERROR_0001_INVALID_ROOT_ELEMENT=\u00c9l\u00e9ment racine non valide.
PentahoSpringBeansConfig.ERROR_0001_INVALID_ROOT_ELEMENT=\u00c9l\u00e9ment racine non valide.
SystemListenersConfig.ERROR_0001_INVALID_ROOT_ELEMENT=\u00c9l\u00e9ment racine non valide.
#
#
#
SolutionRepositoryServiceProxy.ERROR_0001_UNABLE_RETRIEVE_SOLUTION_REPOSITORY=Impossible d'obtenir le contenu du r\u00e9f\u00e9rentiel de la solution.   {0}
#
SchedulerAdminUIComponentProxy.ERROR_0001_PRIVATE_UNABLE_TO_DELETE_JOB=Impossible de supprimer un calendrier priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0002_PRIVATE_UNABLE_TO_EXECUTE_JOB=Impossible d'ex\u00e9cuter un calendrier priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0003_PRIVATE_UNABLE_TO_PAUSE_JOB=Impossible de mettre en pause un calendrier priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0004_PRIVATE_UNABLE_TO_RESUME_JOB=Impossible de reprendre un calendrier priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0005_PRIVATE_UNABLE_TO_RETRIEVE_JOB=Impossible de r\u00e9cup\u00e9rer un calendrier priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0006_PRIVATE_UNABLE_TO_CHECK_IS_PAUSED=Impossible de v\u00e9rifier si un calendrier priv\u00e9 est en pause ou non : {0}
SchedulerAdminUIComponentProxy.ERROR_0007_PRIVATE_UNABLE_TO_RESUME_ALL_JOBS=Impossible de reprendre tous les calendriers priv\u00e9s : {0}
SchedulerAdminUIComponentProxy.ERROR_0008_PRIVATE_UNABLE_TO_PAUSE_ALL_JOBS=\u00ab Impossible de r\u00e9cup\u00e9rer tous les calendriers priv\u00e9s : {0}
SchedulerAdminUIComponentProxy.ERROR_0009_PRIVATE_UNABLE_TO_PAUSE_SHCEDULER=Impossible de mettre en pause un planificateur priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0010_PRIVATE_UNABLE_TO_RESUME_SCHEDULER=Impossible de reprendre un planificateur priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0011_PRIVATE_UNABLE_TO_CREATE_CRON_SCHEDULE=Impossible de cr\u00e9er un calendrier cron priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0012_PRIVATE_UNABLE_TO_UPDATE_CRON_SCHEDULE=Impossible de mettre \u00e0 jour un calendrier cron priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0013_PRIVATE_UNABLE_TO_CREATE_REPEATING_SCHEDULE=Impossible de cr\u00e9er un calendrier r\u00e9p\u00e9titif priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0014_PRIVATE_UNABLE_TO_UPDATE_REPEATING_SCHEDULE=Impossible de mettre \u00e0 jour un calendrier r\u00e9p\u00e9titif priv\u00e9 : {0}
SchedulerAdminUIComponentProxy.ERROR_0015_PRIVATE_UNABLE_TO_RETRIEVE_SHCEDULE_LIST=Impossible de r\u00e9cup\u00e9rer la liste des calendriers priv\u00e9s : {0}
#
HibernateConfigurationService.ERROR_0001_SOLUTION_PATH_IS_NULL=Le chemin d'acc\u00e8s de la solution Pentaho est nul
HibernateConfigurationService.ERROR_0002_NO_CONFIGURATION_FILES_FOUND_IN_PATH=Aucun fichier de configuration Hibernate trouv\u00e9 dans le chemin d'acc\u00e8s
HibernateConfigurationService.ERROR_0003_UNABLE_TO_INITIALIZE_CONFIGURATION=Impossible d'initialiser la configuration
JdbcDriverDiscoveryService.ERROR_0003_UNABLE_TO_INITIALIZE_CONFIGURATION=Impossible d'initialiser la configuration
#
